### DISCUSSION TITLE
Tesla Shanghai

### DISCUSSION THREAD

**Discussion by tonyqus** [shnug/jobs#36] [General]
注：内容不一定准确，仅供参考

Tesla上海主要有2个办公室，一个在临港工厂，一个在浦东世博SK大厦。技术研发岗基本都在临港，朱晓彤的意思是技术研发必须跟着工厂走。

内部加班比较严重，经常996。要求英语沟通，所以对英语口语有相当的要求。

厂区管理很严格，有些区域是不能进的，否则就是违规。甚至有些厂房是不允许带手机进入的，要在门口寄存。

内部的warp drive ERP主要是从SpaceX clone过来的分支版本，用以替代SAP做生产物料和物流管理，是用.NET开发的。里面也有很多go项目。

==========更新===========
现在部分team有名额可以2-3天在SK大厦办公，但每个team只允许少数人过去SK大厦，原则上研发还是跟着工厂走。公司有班车往返临港和SK大厦

**Comment by ccq1** [comment]
佬，请问特斯拉 .net 是 C/S端还是 B/S端啊

**Reply by tonyqus** [reply]
看部门的，ERP是B/S的，不是C/S
